src/domain/synthetics/markets/useMarketsInfoRequest.ts

```diff
@@ -473,8 +485,8 @@ export function useMarketsInfoRequest(chainId: number): MarketsInfoResult {
           openInterestReserveFactorShort: BigNumber.from(
             dataStoreValues.openInterestReserveFactorShort.returnValues[0]
           ),
-          maxOpenInterestLong: BigNumber.from(dataStoreValues.maxOpenInterestLong.returnValues[0]),
-          maxOpenInterestShort: BigNumber.from(dataStoreValues.maxOpenInterestShort.returnValues[0]),
+          maxOpenInterestLong: BigNumber.from(dataStoreValues.maxOpenInterestLong.returnValues[0]).div(marketDivisor),
+          maxOpenInterestShort: BigNumber.from(dataStoreValues.maxOpenInterestShort.returnValues[0]).div(marketDivisor),
```

Comment:
this is also value in config, no need to divide it by divisor

Reply:
fixed